MqttOutput.go

```diff
@@ -0,0 +1,70 @@
+package dnstap
+
+import (
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+	"google.golang.org/protobuf/proto"
+)
+
```

Comment:
This helps clarify our intention that `MqttOutput` implements this interface and provides a small amount of compile-time type checking.
```suggestion

var _ dnstap.Output = (*MqttOutput)(nil)

```